lib/govuk_sidekiq/api_headers.rb

```diff
@@ -0,0 +1,31 @@
+module GOVUKSidekiq
```

Comment:
I think `GovukSidekiq` would be more consistent with other gems. Most apps use `Govuk`: `GovukMessageQueueConsumer` / `GovukContentSchemaTestHelpers` / `GovukAdminTemplate` / `GovukTemplate`.